application/forms/ConfigForm.php

```diff
@@ -72,11 +83,14 @@ public function onSuccess()
         } else {
             return false;
         }
+
+        if (ConfigFormEventsHook::runOnSuccess($this) === false) {
+            Notification::error($this->translate(
+                'Configuration successfully stored despite erroneous module hook(s). See logs for details'
```

Comment:
Sounds unnecessarily vague to me.

Might be more obvious:
> We managed to successfully store your configuration although a module developer failed to provide a perfect hook implementation. See prove for this in the log

Well, on a more serious note:
> Configuration successfully stored. Though, one or more module hooks failed to run. See logs for details

Reply:
I guess I should note: It's really the wording which strikes me here. You surely didn't meant to convey this, but that's how I interpret it. :thinking:

Reply:
The serious note sounds good 😆 Thx!